app/lib/features/chat_ng/widgets/chat_input/chat_editor.dart

```diff
@@ -184,63 +192,59 @@ class _ChatEditorState extends ConsumerState<ChatEditor> {
     chatInputNotifier.emojiPickerVisible(!emojiPickerVisible);
   }
 
-  Widget editorField() {
+  Widget editorField(bool? isEncrypted) {
     final widgetSize = MediaQuery.sizeOf(context);
+    final hintText = isEncrypted.map(
+      (v) => v == true
+          ? L10n.of(context).newEncryptedMessage
+          : L10n.of(context).newMessage,
+      orElse: () => L10n.of(context).newMessage,
+    );
     return Expanded(
-      child: IntrinsicHeight(
```

Comment:
This ```IntrinsicHeight``` wrapper isn't needed as opposed to inner one, so having it removed is recommended as it's expensive.

Reply:
recommended? do you have a reference so I can read up on that?